### PR TITLE
add AMC Score Calculator extension

### PR DESCRIPTION
**Pull request by ryanccn**
## Description

A calculator that **calculates AMC scores** based on user input. Could possibly add AIME and other comps later.

## Type of change

- New extension

## Screenshot

Screen Shot 2021-10-16 at 23 26 21

## Checklist

- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

File: extensions/amc-score-calculator/package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  "name": "amc-score-calculator",
+  "title": "AMC Score Calculator",
+  "description": "Calculates AMC scores for 8, 10, and 12",
+  "icon": "command-icon.png",
+  "author": "ryanccn",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "eight",
+      "title": "AMC 8",
```
Comment (PitNikola) [inline extensions/amc-score-calculator/package.json:11]:
```suggestion
      "title": "Calculate AMC 8 Score",
```

File: extensions/amc-score-calculator/package.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      "subtitle": "",
+      "description": "Calculate AMC 8 scores",
+      "mode": "view"
+    },
+    {
+      "name": "ten",
+      "title": "AMC 10",
```
Comment (PitNikola) [inline extensions/amc-score-calculator/package.json:18]:
```suggestion
      "title": "Calculate AMC 10 Score",
```

File: extensions/amc-score-calculator/package.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      "subtitle": "",
+      "description": "Calculate AMC 10 scores",
+      "mode": "view"
+    },
+    {
+      "name": "twelve",
+      "title": "AMC 12",
```
Comment (PitNikola) [inline extensions/amc-score-calculator/package.json:25]:
```suggestion
      "title": "Calculate AMC 12 Score",
```

File: extensions/amc-score-calculator/package.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  "description": "Calculates AMC scores for 8, 10, and 12",
+  "icon": "command-icon.png",
+  "author": "ryanccn",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "eight",
+      "title": "AMC 8",
+      "subtitle": "",
+      "description": "Calculate AMC 8 scores",
```
Comment (PitNikola) [inline extensions/amc-score-calculator/package.json:13]: Maybe you can provide more detailed description, something like "Get your AMC 8 score from number of correct answers"

Comment (PitNikola) [reply extensions/amc-score-calculator/package.json:13]: Same in other two commands

File: extensions/amc-score-calculator/src/eight.tsx [hunk 5]
```diff
@@ -0,0 +1,5 @@
+import Generic from './generic';
+
+export default () => {
+  return <Generic title={'AMC 8 Score Calculator'} correctValue={1} emptyValue={0} />;
```
Comment (PitNikola) [inline extensions/amc-score-calculator/src/eight.tsx:4]: So does AMC 8 always returns the same number as number of correct answers? What's the value in this calculation? Or am I missing something?

**Comment by ryanccn** [comment]
Got it, will edit later this week :+1:

**Comment by PitNikola** [comment]
@ryanccn, I'm gonna close this PR for now. Please feel free to re-open once you have some changes to review 👍

**Pull request closed without merging**